consul/tests/test_unit.py

```diff
@@ -320,6 +320,10 @@ def test_cull_services_list():
     consul_check.service_whitelist = ['service_1', 'service_2', 'service_3']
     assert set(consul_check._cull_services_list(services)) == set(consul_check.service_whitelist)
 
+    # Excluded services will not be in final service list
```

Comment:
Could we also test this use case and assert that exclude list is used and not whitelist?
https://github.com/DataDog/integrations-core/blob/249e4d69e61939711db31beadd197bbc04e2a27f/consul/datadog_checks/consul/consul.py#L265-L270


Can we also test max_services against a services_exclude > max_services? More test cases to cover

Reply:
submitted additional tests to cover more cases